src/explore-education-statistics-common/src/components/Details.tsx

```diff
@@ -141,7 +141,9 @@ const Details = ({
           data-testid={formatTestId(`Expand Details Section ${summary}`)}
         >
           {summary}
-          {hiddenText && <VisuallyHidden> {hiddenText}</VisuallyHidden>}
+          {hiddenText && (
+            <VisuallyHidden>{`${' '} ${hiddenText}`}</VisuallyHidden>
```

Comment:
This is a template string so there's no need to interpolate a string into it.  Additionally, this is placing 2 spaces rather than one. The following would be sufficient:

```tsx
<VisuallyHidden>{` ${hiddenText}`}</VisuallyHidden>
```